### DISCUSSION TITLE
Has Public Access been broken?

### DISCUSSION THREAD

**Discussion by Marcbacca** [mealie-recipes/mealie#3966] [General]
I'm pretty sure public access used to work...that's a big reason I use Mealie. I have a group that is public and has 'allow users outside your group' enabled. My recipes are also all public, but I am always prompted to login when I try to access Mealie instead of being passed through in a read-only manner. Is this something which was recently broken? ... or is it me? Thanks.

I should mention I'm running this through Unraid, v1.11.0 with sqlite
Everything required in the docs are met: https://docs.mealie.io/documentation/getting-started/usage/permissions-and-public-access/#public-recipe-access

**Comment by michael-genson** [comment]
Public access still works fine, so I would double check your permissions. Make sure:
- your group is _not_ private
- your group's "Allow users outside of your group to see your recipes" preference is set

Assuming those are both set correctly, check if you have multiple groups. If you do, only the non-private group can be navigated to. If it's not the default group, you have to go there explicitly (e.g. `https://your-site-url.com/g/your-group-slug`)

**Reply by Marcbacca** [reply]
Hi, thanks for the reply. It's the only/default group, and those settings are setup. If I hit https://mealie.mydomain.com .. the /login path is automatically appended. However, https://mealie.mydomain.com/g/home ... this works in a private window without a login or anything, very strange. I am now wonder if this is an argo tunnel/cloudflared setup issue. But thanks for the info!

**Reply by michael-genson** [reply]
Can you try renaming your group to "home"? (lowercase h). Someone raised a similar issue a bit ago, there might be something wrong with the redirect logic.

**Reply by Marcbacca** [reply]
Yup, you are a God among us mortal men lol. Renaming the group home works, Home does not work. There is definitely an issue with the redirect logic, cases don't match or something... I am certainly not a programmer. Thanks again for the help! .... this is fantastic.
